log/log.go

```diff
@@ -330,6 +330,19 @@ func (l *Log) GetBoleto(content interface{}, msgType string) {
 	})()
 }
 
+//handleMaskErros Handle de erro para funções internas para garantir que, caso ocorra algum problema interno
+//o conteúdo continuará sendo enviado, porém sem a devida máscara.
+
+func (l *Log) HandleMaskErrors(method string, messageType string, methodContent interface{}) {
+	if r := recover(); r != nil {
+		errorDescription := fmt.Sprintf("Recovering from panic during %s operation", method)
+		logMessage := fmt.Sprintf("to {BankName} (%s) | {Recipient} - %s", messageType, errorDescription)
+		recoverError := fmt.Errorf("%s", r)
```

Comment:
opa, aqui faltou o **%v**, que é um tipo mais genérico, visto que o **r** é **interface{}**
```suggestion
		recoverError := fmt.Errorf("%v", r)
```